habitat/plan.ps1

```diff
@@ -21,7 +21,6 @@ $pkg_deps=@(
   "chef/chef-powershell-shim"
   "core/visual-cpp-redist-2015"
 )
-$pkg_build_deps=@( "core/git")
```

Comment:
This is removed because @mwrock figured out that this dep was installing an _additional_ bash which was causing build failures in Windows. Git was already included as part of msys2, so this isn't necessary, and breaks things.